src/lvgl_private.h

```diff
@@ -2,6 +2,12 @@
  * @file lvgl_private.h
  *
  */
+ 
+/*
+ *                            DO NOT EDIT
```

Comment:
```suggestion
 *  DO NOT EDIT
```

Reply:
I'm ok with adding this PR :slightly_smiling_face:

Reply:
OK then I will fix the formatting error.